### ISSUE TITLE
[v5] Lint more files?

### ISSUE THREAD

**Issue by pdehaan** [mozilla/shield-studies-addon-utils#137]
Currently we're only linting the `./webExtensionApis/**/*.{jsm,js,json}` files (per #134 PR).

https://github.com/mozilla/shield-studies-addon-utils/blob/aa1f884ed660145a1603b3e071d2b96264c360e5/package.json#L51

Not sure about all the other files (like `./examples/**`, `./misc/**`, `./testUtils/*`, and root files. It feels like we should be linting those as well so nothing slips through the cracks.
But I'm not sure how ./misc/shield-study-helper-addon/* works since it almost looks and smells like some submodule of sorts (nested package.json and package-lock.json up in this monorepo).

```sh
$ git ls-files | egrep "\.(js|jsm|json)$"

.eslintrc.js
package-lock.json
package.json

examples/test-addon/src/background.js
examples/test-addon/src/extension-page-for-tests/page.js
examples/test-addon/src/manifest.json
examples/test-addon/src/studySetup.js
examples/test-addon/test/functional/shield_utils_test.js
examples/test-addon/test/functional/utils.js
examples/test-addon/web-ext-config.js

misc/shield-study-helper-addon/addon/bootstrap.js
misc/shield-study-helper-addon/addon/webextension/.eslintrc.json
misc/shield-study-helper-addon/addon/webextension/manifest.json
misc/shield-study-helper-addon/addon/webextension/qa.js
misc/shield-study-helper-addon/package-lock.json
misc/shield-study-helper-addon/package.json
misc/shield-study-helper-addon/run-firefox.js

testUtils/executeJs.js
testUtils/nav.js
testUtils/pings.js
testUtils/setup.js
testUtils/ui.js
testUtils/wip.js

webExtensionApis/prefs/api.js
webExtensionApis/prefs/schema.json
webExtensionApis/study/schema.json
webExtensionApis/study/src/index.js
webExtensionApis/study/src/jsonschema.js
webExtensionApis/study/src/sampling.js
webExtensionApis/study/src/schemas/index.js
webExtensionApis/study/src/schemas/schema.studySetup.json
webExtensionApis/study/src/schemas/schema.weightedVariations.json
webExtensionApis/study/src/studyUtils.js
webExtensionApis/study/src/studyUtilsBootstrap.js
webExtensionApis/study/webpack.config.js
```

**Comment by motin** [comment]
I am pro for linting `./**/*.{jsm,js,json}`

> But I'm not sure how ./misc/shield-study-helper-addon/* works since it almost looks and smells like some submodule of sorts (nested package.json and package-lock.json up in this monorepo).

Yes, it is a separate add-on in this monorepo. In https://github.com/mozilla/shield-studies-addon-utils/issues/133 I made it lint anyhow, so that we can keep the policy of keeping everything in this monorepo linted.

**Comment by motin** [comment]
@gregglind is most likely in favor of linting everything as well [Linting rules should be totally righteous for webExtensions and json everywhere in this. #131](https://github.com/mozilla/shield-studies-addon-utils/issues/131) :)

**Comment by gregglind** [comment]
I am 100% for linting everything.

**Comment by motin** [comment]
This was fixed in #152
